### ISSUE TITLE
Want to download entire event log

### ISSUE THREAD

**Issue by gordyvision** [HearthSim/Joust#252]
Is there a way to download/export the entire event log that is in plain english? I have looked at the hearthstone python modules and haven't found a nice way to convert a hsreplay xml to plain english, the modules look to be more focused on converting power.log to XML. Joust seems to generate this event log really well but I'm not seeing a way to get that output on its own (haven't coded in React/TypeScript before though). Could probably make my own python module to do a similar xml conversion but if something already exists it would be nice to just use that.

Thanks in advance and sorry if I missed this info somewhere.

**Comment by beheh** [comment]
Hi there! I believe creating a full textual log of what happened in a Hearthstone game is a fairly complex task, because the log is so verbose. You can already see this when checking Joust's normal event log viewer, there will be lots of uninteresting and non-sensical changes. While we welcome contributions to this in Joust, we are not prioritizing that ourselves for now, and we don't see the need to offer a download functionality of the log as a feature of Joust.

You can see Joust's existing logic over at https://github.com/HearthSim/Joust/blob/a166ad4b29c6c425605b27725467ed6df50e9383/ts/components/EventLog.tsx to get an idea of the types of heuristics that are necessary.

As this is not in the scope, I'm closing this issue for now, but feel free to continue posting in this issue if you have any further questions.
